### PR TITLE
Add Trish Scarborough

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tic/tic.json [hunk 1]
```diff
@@ -1,122 +1,187 @@
...
+		"position": 2,
+		"quantity": 1,
+		"sanity": 7,
+		"skill_agility": 2,
+		"skill_intellect": 2,
+		"skill_combat": 2,
+		"skill_willpower": 2,
+		"subname": "The Student",
+		"text": "<b>Forced</b> - At the start of the investigation phase: Draw 1 card. Discard the card beneath Amanda Sharpe. Choose a card from your hand and place it beneath her.\n[reaction] At the start of a skill test you are performing: Commit the card beneath Amanda Sharpe, if able. Do not discard it when the test ends.\n[elder_sign] effect: +0. For this test, you may double the number of skill icons on the card beneath Amanda Sharpe.",
+		"traits": "Miskatonic. Scholar",
```
Comment (drawntotheflame) [inline pack/tic/tic.json:58]: Period missing after 'Scholar.'

File: pack/tic/tic.json [hunk 2]
```diff
@@ -1,122 +1,187 @@
...
+		"type_code": "investigator"
+	},
+	{
+		"back_flavor": "Trish had always excelled in both body and mind, her unwavering focus on excellence and her fearless disposition a winning combination. Given her stellar academic and athletic records, everyone expected to see her become something great. Instead, Trish took a job at a commercial code company. The initial shock of her humble work eventually wore off, and the exceptional Trish Scorborough faded into obscurity. This suited Trish just fine, as it made it harder for anyone to realize the truth: her job was a cover for her true work as a spy with the Black Chamber, the Bureau's code-breaking agency.",
```
Comment (drawntotheflame) [inline pack/tic/tic.json:62]: typo: 'exceptional Trish Scorborough' -> 'exceptional Trish Scarborough'

File: pack/tic/tic.json [hunk 3]
```diff
@@ -1,122 +1,187 @@
...
+		"back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Guardian cards ([guardian]) level 0-5, Neutral cards level 0-5, Mystic cards ([mystic]) level 0-2.\n<b>Deckbuilding Requirements</b> (do not count toward deck size): In the Shadows, Shadow Agents, 1 random basic weakness.",
```
Comment (drawntotheflame) [inline pack/tic/tic.json:63]: Her cards are 'Rogue' and 'Seeker'.

**Comment by drawntotheflame** [comment]
@zzorba when you've done these tweaks, do ping Kam to put this live. Thanks.

**Comment by zzorba** [comment]
All fixed, @Kamalisk this can be merged whenever.

**Pull request closed without merging**